### PR TITLE
update release notes

### PR DESCRIPTION
**Pull request by cbudz**
Adding recent 1.2.z updates to the release notes.

### CODE REVIEW

**Comment by cbudz** [comment]
@TarikFD can you take a look when you have a second?

File: release-notes/topics/hub-426.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[[hub-426-intro]]
+= Automation Hub 4.2.6
+
+Automation Hub allows you to discover and utilize new certified automation content from Red Hat Ansible and Certified Partners. On Ansible Automation Hub, you can discover and manage Ansible Collections which is supported automation content developed by both partners and Red Hat for use cases such as cloud automation, network automation, security automation, and more.
+
+.Automation Hub bug fixes
+
+* Implemented filters for state and keywords on imports API.
```
Comment (TarikFD) [inline release-notes/topics/hub-426.adoc:8]:
```suggestion
* Implemented filters for state and keywords on imports API
```

File: release-notes/topics/hub-426.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+* Create 'inbound-namespaces' whenever a namespace is created.
```
Comment (TarikFD) [inline release-notes/topics/hub-426.adoc:9]:
```suggestion
* Create 'inbound-namespaces' whenever a namespace is created
```

**Review by TarikFD — CHANGES_REQUESTED** [review]
Some very minor punctuation edits for consistency. Everything else looks good. I can merge once the changes are made

**Review by TarikFD — APPROVED** [review]
Looks good. Merging

**Pull request merged**